daprdocs/content/en/go-sdk-docs/_index.md

```diff
@@ -0,0 +1,155 @@
+---
+type: docs
+title: "Getting started with the Dapr pluggable components Go SDK"
+linkTitle: "Go"
+weight: 1000
+description: How to get up and running with the Dapr pluggable components Go SDK
+no_list: true
+is_preview: true
+---
+
+Dapr offers packages to help with the development of Go pluggable components.
+
+## Prerequisites
+
+- [Go 1.19](https://go.dev/dl/) or later
+- [Dapr 1.9 CLI]({{< ref install-dapr-cli.md >}}) or later
+- Initialized [Dapr environment]({{< ref install-dapr-selfhost.md >}})
+- Linux, Mac, or Windows (with WSL)
+
+{{% alert title="Note" color="primary" %}}
+Development of Dapr pluggable components on Windows requires WSL as some development platforms do not fully support Unix Domain Sockets on "native" Windows.
+{{% /alert %}}
+
+## Application creation
+
+Creating a pluggable component starts with an empty Go application.
+
+```bash
+mkdir example
+cd component
+go mod init example
+```
+
+## Import Dapr packages
+
+Import the Dapr pluggable components SDK package.
+
+```bash
+go get github.com/dapr-sandbox/components-go-sdk@v0.1.0
+```
+
+## Create main package
+
+In `main.go`, import the Dapr plugggable components package and run the application.
+
+```go
+package main
+
+import (
+	dapr "github.com/dapr-sandbox/components-go-sdk"
+)
+
+func main() {
+	dapr.MustRun()
+}
+```
+
+This creates an application with no components. You will need to implement and register one or more components.
+
+## Implement and register components
+
+ - [Implementing an input/output binding component]({{< ref go-bindings >}})
+ - [Implementing a pub/sub component]({{< ref go-pub-sub >}})
+ - [Implementing a state store component]({{< ref go-state-store >}})
+
+{{% alert title="Note" color="primary" %}}
+Only a single component of each type can be registered with an individual service. However, [multiple components of the same type can be spread across multiple services]({{< ref go-advanced >}}).
+{{% /alert %}}
+
+## Test components locally
+
+Pluggable components can be tested by starting the application on the command line and configuring a Dapr sidecar to use it.
+
+To start the component, in the application directory:
```

Comment:
I think we need to mention the creation of the `/tmp/dapr-components-sockets` directory somewhere. This is required and without this, the steps described here will fail, no?

How about:
```suggestion

### Create a directory to hold your UDS

Dapr and your components will communicate using UDS files. By default, Dapr looks for those files in a directory named `/tmp/dapr-components-sockets`. If this directory doesn't exist, create it. Your component will also default to creating its Unix Domain Socket files in this diretory.

### Start your application

Pluggable components can be tested by starting the application on the command line and configuring a Dapr sidecar to use it.

To start the component, in the application directory:
```